packages/lib-user/src/components/UserStats/components/ProfileHeader.spec.js

```diff
@@ -0,0 +1,87 @@
+import { render, screen } from '@testing-library/react'
+import { composeStory } from '@storybook/react'
+
+import Meta, { Default, Group, GroupWithHours, UserWithHours } from './ProfileHeader.stories.js'
+
+describe('components > UserStats > ProfileHeader', function () {
+  describe('with a user', function () {
+    it('should show the user\'s avatar', function () {
+      const DefaultStory = composeStory(Default, Meta)
+      render(<DefaultStory />)
```

Comment:
```suggestion
  describe('with a user', function () {
    const DefaultStory = composeStory(Default, Meta)

    it('should show the user\'s avatar', function () {
      render(<DefaultStory />)
```

Recommendation to define `DefaultStory` once in this `describe()` block so that it can be reused in each `it()`. All the other `DefaultStory` statements can be deleted.